test/public/createUploadLink.test.js

```diff
@@ -654,6 +656,42 @@ module.exports = (tests) => {
     }
   );
 
+  tests.add('`createUploadLink` with `execute`', async () => {
+    let fetchUri;
+    let fetchOptions;
+
+    const revertGlobals = revertableGlobals({
+      async fetch(uri, options) {
```

Comment:
For this test, `fetch` is better as a `createUploadLink` option to avoid messing with globals.